SQL_module/SubjectForm_Task2/db_manipulation.php

```diff
@@ -0,0 +1,100 @@
+<?php
+
+    include "./db_connection.php";
+
+    $table_name = "electives";
+
+    function insertInTable($title, $description, $lecturer){
+        try {
+
+            $connection = db_connection();
+            $sql = "INSERT INTO electives(title,description,lecturer) VALUES (:title, :description, :lecturer)";
+    
+            $preparedSql = $connection->prepare($sql) or die("Failed to prepare sql query.");
+            $preparedSql->bindParam(':title', $title);
+            $preparedSql->bindParam(':description', $description);
+            $preparedSql->bindParam(':lecturer', $lecturer);
+            
+            $preparedSql->execute() or die("Filed to execute sql query.");   
+        }
+        catch(PDOException $error) {
+            echo $error->getMessage();
+        }
+    }
+
+    function printContentOfTable(){
+        
+        global $table_name;
+        try {
+            $connection = db_connection();
+            $sql = "SELECT * FROM $table_name";
+            
+            $result = $connection->prepare($sql) or die("Filed to prepare sql query.");
+            $result->execute() or die("Filed to execute sql query.");
```

Comment:
Typo - failed. It is on multiple places.
```suggestion
            $result->execute() or die("Failed to execute sql query.");
```